test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -980,6 +982,10 @@ def _revert_pbsconf_comm(self, primary_server, vals_to_set):
                 if scppath != "scp":
                     new_pbsconf["PBS_SCP"] = scppath
                     restart_comm = True
+            if "PBS_LOG_HIGHRES_TIMESTAMP" not in new_pbsconf  or \
```

Comment:
PBS_LOG_HIGHRES_TIMESTAMP will always be in new_pbsconf, so you don't need "if "PBS_LOG_HIGHRES_TIMESTAMP" not in new_pbsconf" check. Same comment for other functions.